discovery/src/main/java/com/alibaba/fescar/discovery/registry/EurekaRegistryServiceImpl.java

```diff
@@ -143,7 +143,7 @@ public void onEvent(EurekaEvent event) {
             });
         }
 
-        return Lists.newArrayList(CLUSTER_ADDRESS_MAP.get(clusterName));
+        return Lists.newArrayList(CLUSTER_ADDRESS_MAP.get(clusterName.toUpperCase()));
```

Comment:
1.better to add  toUpperCase() for clusterName when put 
`CLUSTER_ADDRESS_MAP.put(app.getName().toUpperCase(), addressSet);`
2.i have a question ,the Application 's name can be null or empty ?